r2i/onnxrt/engine.h

```diff
@@ -50,19 +64,27 @@ class Engine : public IEngine {
   size_t output_size;
   size_t num_input_nodes;
   size_t num_output_nodes;
+  Ort::Env env {nullptr};
+  Ort::SessionOptions session_options {nullptr};
+  std::shared_ptr<Ort::Session> session;
```

Comment:
I'm confused why you only treat the session as a smart pointer and the rest as static members. If possible, I'd prefer to treat all of them as static members.